docs/refactoring/sonarcloud-report.md

```
@@ -0,0 +1,24 @@
+
+## До: 
+![](sonar-before1.png)
+
+### Трошки почистивши False Positive's, маємо наступне:
+![](sonar-before2.png)
+
+## Після:
+![](sonar-after.png)
+
+## Обрані метрики:
+| Метрика           | До    | Після |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)